warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -489,6 +496,27 @@ public void run() {
         log.info("Bulk map file loader shutting down.");
     }
     
+    private void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
+        for (int i = 0; i < cleanupDirectories.length; i++) {
+            markSourceFilesLoaded(cleanupDirectories[i]);
```

Comment:
I think we may have issues here if we have multiple map loaders working against the same warehouse.  Perhaps   this should attempt the "grab control" of the directory by renaming the cleanup file and only when successfully doing that will actually implement the cleanup.

Reply:
I believe I made changes that address this. Please let me know what you think.

Reply:
think this is covered with the REMOVE_FILE_MARKER transition